src/_listBase.scss

```diff
@@ -302,4 +324,21 @@
   &--has-message {
     padding-top: 2rem;
   }
+
+  &__counter {
+    font-size: 0.875rem;
+    color: var(--sapContent_MarkerTextColor);
+    padding-left: $fd-list-item-padding-x;
+    text-shadow: none;
+    flex: none;
+
+    @include fd-active() {
+      color: var(--sapHighlightTextColor);
+    }
+  }
```

Comment:
I think we already have a counter.

Reply:
i have removed all the duplicate classes which was coming from platform list. i have verified Ngx as well using correct styles classes.